virtualbox/vbm.go

```diff
@@ -99,6 +100,11 @@ func getHostOnlyNetworkInterface(mc *driver.MachineConfig) (string, error) {
 
 	for _, n := range nets {
 		if dhcp, ok := dhcps[n.NetworkName]; ok {
+            fmt.Printf("IPv4 / DHCP IP: %s == %s\n", dhcp.IPv4.IP, mc.DHCPIP)
```

Comment:
Well, it's essential for how I'm working with this. For some reason, I didn't have any proper host-only networks. (That might be because I upgraded from VBox 4 / 5rc1 on this machine, when host-only networks didn't work yet.)
So I personally need these print statements (or a written list somewhere) to know the exact values I need to create the host-only network that boot2docker-cli tries to find.

I'm not sure what's really supposed to happen here. I guess boot2docker-cli should automatically create the host-only network it needs? Maybe we just need to fix that for VBox 5.